test/cron-schedule.test.ts

```diff
@@ -0,0 +1,52 @@
+import { CronSchedule } from '../src';
+
+describe('CronSchedule', () => {
+  test('cron expression for running every minute', () => {
+    expect('* * * * ?').toEqual(CronSchedule.everyMinute().expressionString);
+  });
+
+  test('cron expression for running every hour', () => {
+    expect('0 * * * ?').toEqual(CronSchedule.hourly().expressionString);
+  });
+
+  test('cron expression for running every day', () => {
+    expect('0 0 * * ?').toEqual(CronSchedule.daily().expressionString);
+  });
+
+  test('cron expression for running every week', () => {
+    expect('0 0 ? * 0').toEqual(CronSchedule.weekly().expressionString);
+  });
+
+  test('cron expression for running every month', () => {
+    expect('0 0 1 * ?').toEqual(CronSchedule.monthly().expressionString);
+  });
+
+  test('cron expression for running every year', () => {
+    expect('0 0 1 1 ?').toEqual(CronSchedule.annually().expressionString);
+  });
+
+  test('cron expressions day and dow are mutex: given weekday', () => {
+    // Run every 10 minutes Monday through Friday
+    expect('0/10 * ? * MON-FRI').toEqual(CronSchedule.custom({
```

Comment:
Replace the the order, the expected result should be passed to `toEqual`, not the actual.

Reply:
Right, will make the update.